README.md

```diff
@@ -226,8 +227,16 @@ Besides `esm.pretrained.esmfold_v1()` which is the best performing model we reco
 also provide `esm.pretrained.esmfold_v0()` which was used for the experiments in
 [Lin et al. 2022](https://doi.org/10.1101/2022.07.20.500902).
 
-We also provide a script (`scripts/esmfold_inference.py`) that efficiently predicts structures in bulk from a FASTA file using ESMFold. This can be run with
-
+We also provide a script (`esm-fold` or `scripts/esmfold_inference.py`) that efficiently predicts structures in bulk from a FASTA file using ESMFold. This can be run with one of the following commands:
+```bash
```

Comment:
We can just document the CLI version and remove the `python scripts/esmfold_inference.py` invocation.

Reply:
Done!